CMSgov/eAPD
# Tbolt/2675 rich text editor lables

Resolves #2675 

### Description
Adds more descriptive labels to the Rich Text editor. See original ticket for full details and backstory.

### Significant changes or possible side effects
No significant changes or possible side effects as far as I know

### Automated test cases written
Snapshots updated

### Steps to manually verify this change

1. Login and view an APD
2. Use a screen reader
3. Verify that when focusing/selecting a rich text editor field, a descriptive title is read

### This pull request is ready to review when

      tests, refactor
      screenreader, text scaling) OR an exemption is documented

### This pull request can be merged when

- [ ] QA has verified the accessibility using tools such as screen readers and
      WAVE (accessibility evaluation tool)
